test/e2e/tests/identity/account-syncing/multi-srp.spec.ts

```diff
@@ -107,22 +105,14 @@ describe('Account syncing - Multiple SRPs', function () {
         // Import second SRP (this will automatically create the third account)
         await accountListPage.startImportSecretPhrase(
           IDENTITY_TEAM_SEED_PHRASE_2,
-          {
-            isMultichainAccountsState2Enabled: true,
-          },
         );
 
         // Importing an SRP can be long, so we add a bit of extra time here
         await driver.delay(10000);
 
-        // Wait for the import to complete and sync
-        await waitUntilSyncedAccountsNumberEquals(3);
```

Comment:
To be reworked in a follow-up PR: We should probably re-enable this, and if this is not working anymore, then we have to re-adapt the values most probably, but this seems important for B&S coverage!